templates/registration/logout.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+{% block content %}
+  <p>Logged out</p>
+  <a href="{% url "taxi:login" %}">Click here to login again</a>
```

Comment:
Inconsistent use of quotation marks in the template tag. Consider using single quotes consistently for URL template tags, like `{% url 'taxi:login' %}`.